L1Trigger/L1THGCal/src/backend/HGCalMulticlusteringImpl.cc

```diff
@@ -170,6 +192,314 @@ void HGCalMulticlusteringImpl::clusterizeDBSCAN( const std::vector<edm::Ptr<l1t:
 }
 
 
+
+
+
+std::map<std::vector<int>,float> HGCalMulticlusteringImpl::fillHistoClusters( const std::vector<edm::Ptr<l1t::HGCalCluster>> & clustersPtrs ){
+
+
+    std::map<std::vector<int>,float> histoClusters; //key[0] = z.side(), key[1] = bin_R, key[2] = bin_phi
+
+    for(std::vector<edm::Ptr<l1t::HGCalCluster>>::const_iterator clu = clustersPtrs.begin(); clu != clustersPtrs.end(); ++clu){
```

Comment:
Could you use a range-based for loop? And so avoid`(**clu).` (replaced with `clu->`). I find it difficult to read.